internal/command/deploy/common_secrets.go

```diff
@@ -0,0 +1,4 @@
+package deploy
+
+// The top 100 most common secrets
+var commonSecrets = []string{"SECRET_KEY_BASE", "SU_PASSWORD", "REPL_PASSWORD", "OPERATOR_PASSWORD", "DATABASE_URL", "FLY_CONSUL_URL", "APOLLO_KEY", "APOLLO_GRAPH_REF", "APOLLO_USAGE_REPORTING_INGRESS_URL", "APOLLO_ENDPOINT", "CLOUD_ROUTER", "APOLLO_ROUTER_CONFIG_ENV_PREFIX", "FLY_API_TOKEN", "SSH_KEY", "SSH_CERT", "SESSION_SECRET", "APP_KEY", "RAILS_MASTER_KEY", "REDIS_URL", "SECRET_KEY", "TOKEN_MANAGER_PUBLIC_KEY", "AWS_SECRET_ACCESS_KEY", "LIVEBOOK_PASSWORD", "AWS_ACCESS_KEY_ID", "SENDGRID_API_KEY", "LIVEBOOK_SECRET_KEY_BASE", "MONGODB_URI", "IKU_TURSO_AUTH_JWT_PUBLIC_KEY", "JWT_SECRET", "PORT", "DB_PASSWORD", "REDIS_PASSWORD", "DB_USERNAME", "NODE_ENV", "OPENAI_API_KEY", "TEMPORAL_CLUSTER_URL", "SENTRY_DSN", "PAPERTRAIL_API_TOKEN", "POSTGRES_PASSWORD", "MESSAGEBIRD_API_KEY", "FLY_RESTORED_FROM", "SHOPIFY_API_SECRET", "API_SECRET", "DB_DATABASE", "DOPPLER_TOKEN", "MYSQL_PASSWORD", "WG_BUILDER_PROJECT_ID", "WG_BUILDER_FLY_API_TOKEN", "WG_BUILDER_TEMPORAL_NAMESPACE", "WG_BUILDER_RELEASE", "NO_SERVICE_FROM", "NO_SERVICE_TO", "MYSQL_ROOT_PASSWORD", "ENABLE", "ENAIA_HOSTNAME", "THEME", "TIME_FORMAT", "DISPLAY_UNITS", "BG_HIGH", "CUSTOM_TITLE", "BG_LOW", "ALARM_HIGH", "ALARM_LOW", "GOOGLE_CLIENT_ID", "NIGHT_MODE", "BG_TARGET_BOTTOM", "BG_TARGET_TOP", "ALARM_URGENT_HIGH", "ALARM_TYPES", "ALARM_TIMEAGO_URGENT", "ALARM_TIMEAGO_WARN", "ALARM_URGENT_LOW", "ALARM_TIMEAGO_WARN_MINS", "ALARM_TIMEAGO_URGENT_MINS", "ENAIA_ALGOLIA_ACCOUNTS_AND_DEALS_INDEX_NAME", "SHOW_RAWBG", "STRIPE_SECRET_KEY", "PASSWORD", "BRIDGE_PASSWORD", "BRIDGE_USER_NAME", "ROUTER_GRAPH_REF", "GCP_ACCOUNT_EMAIL", "GCP_ACCOUNT_PRIVATE_KEY", "LOGFLARE_API_KEY", "SHOW_PLUGINS", "AWS_ACCESS_KEY", "DB_HOST", "GOOGLE_CLIENT_SECRET", "BRIDGE_SERVER", "TAILSCALE_AUTHKEY", "AWS_SECRET_KEY", "WG_OTEL_AUTH_TOKEN", "DOPPLER_CONFIG", "MMCONNECT_SERVER", "DOPPLER_ENVIRONMENT", "MMCONNECT_USER_NAME", "DOPPLER_PROJECT", "MMCONNECT_PASSWORD", "ACCESS_TOKEN"}
```

Comment:
We should do a manual 👀 through this list - some of these are probably false positives. 
`TIME_FORMAT` for example seems safe.

Reply:
Gotcha, I think the current list should be better